### PR TITLE
[CDAP-17924] Redesign namespace admin

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://cdap.atlassian.net/browse/CDAP-17924

### CODE REVIEW

File: app/cdap/components/NamespaceAdmin/ComputeProfiles.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      display: 'none',
+    },
+    inputLabel: {
+      marginBottom: 0,
+      cursor: 'inherit',
+    },
+  };
+});
+
+const ComputeProfiles: React.FC = () => {
```
Comment (ajainarayanan) [inline app/cdap/components/NamespaceAdmin/ComputeProfiles.tsx:43]: nit: If we don't have any props do we need the React.FC type here?

File: app/cdap/components/NamespaceAdmin/Preferences.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  }
+
+  return (
+    <div>
+      <div className={classes.subtitleSection}>
+        <Button color="primary" variant="contained" onClick={toggleModal}>
+          Edit
+        </Button>
+      </div>
+      <SortableTable
```
Comment (ajainarayanan) [inline app/cdap/components/NamespaceAdmin/Preferences.tsx]: Can we use the `Table` that we introduced recently for consistency? May be modify `SortableTable` to use the new components, but feel that might be a bigger change.

**Review by ajainarayanan — APPROVED** [review]
Minor comments 👍 LGTM

File: app/cdap/components/NamespaceAdmin/ComputeProfiles.tsx (hunk 1 above)
Comment (elfenheart) [reply app/cdap/components/NamespaceAdmin/ComputeProfiles.tsx:43]: it's to establish the exported type as a react component instead of normal function

**Pull request closed without merging**